helix-term/src/keymap.rs

```diff
@@ -278,6 +296,23 @@ pub fn default() -> Keymaps {
             key!('k') => commands::extend_line_up,
             key!('l') => commands::extend_char_right,
 
+            KeyEvent {
+                code: KeyCode::Left,
+                modifiers: KeyModifiers::NONE
+            } => commands::move_char_left as Command,
```

Comment:
You should use the `extend_` methods here

Reply:
My bad